vscode/src/chat/MessageProvider.ts

```diff
@@ -213,7 +213,15 @@ export abstract class MessageProvider extends MessageHandler implements vscode.D
             },
             onTurnComplete: async () => {
                 typewriter.close()
-                await typewriter.finished
+                typewriter.stop()
+
+                const formattedBotMessage = reformatBotMessage(text, responsePrefix)
+                if (!formattedBotMessage) {
```

Comment:
```suggestion
                if (!text) {
```
Would text works better in case responsePrefix is not empty but text is?

Reply:
good point, will update